data-processing-lib/python/src/data_processing/test_support/abstract_test.py

```diff
@@ -35,15 +35,17 @@ def get_tables_in_folder(dir: str) -> list[pa.Table]:
     return [TransformUtils.convert_binary_to_arrow(data) for data in files.values()]
 
 
-def get_files_in_folder(dir: str, ext: str) -> dict[str, bytes]:
+def get_files_in_folder(dir: str, ext: str, return_data: bool = True) -> dict[str, bytes]:
     """
     Get  list of Tables loaded from the parquet files in the given directory.  The returned
     list is sorted lexigraphically by the name of the file.
-    :param dir:
+    :param dir: directory
+    :param ext: extension
+    :param return_data: flag to return data - default True
```

Comment:
language problem.  I would say you're returning the file names not the data.  and also, above you mention `parquet`.  probably mean files matching the extension now.

Reply:
so, return_file_names instead of return_data.

Reply:
descriptive and meaningful variable names means you can make short cuts on documentation, as above :).

Reply:
Actually, you got it wrong we always return file names, but if this flag is set we return actual data, so it seems correct